cmd/harvest/harvest.go

```diff
@@ -106,8 +113,7 @@ func doManageCmd(cmd *cobra.Command, args []string) {
 
 	//cmd.DebugFlags()  // uncomment to print flags
 
-	pollers, err := conf.GetPollers(opts.config)
-	if err != nil {
+	if pollers, err = conf.GetPollers2(opts.config); err != nil {
```

Comment:
do we want the pollers returned in the same order that they're listed in the harvest.yml? I think we do for deterministic printing. If so, we can use 
`conf.Config.PollersOrdered`